recipes-bsp/opensbi/opensbi_git.bb

```diff
@@ -0,0 +1,50 @@
+SUMMARY = "RISC-V Open Source Supervisor Binary Interface (OpenSBI)"
+DESCRIPTION = "OpenSBI aims to provide an open-source and extensible implementation of the RISC-V SBI specification for a platform specific firmware (M-mode) and a general purpose OS, hypervisor or bootloader (S-mode or HS-mode). OpenSBI implementation can be easily extended by RISC-V platform or System-on-Chip vendors to fit a particular hadware configuration."
+LICENSE = "BSD-2-Clause"
+LIC_FILES_CHKSUM = "file://COPYING.BSD;md5=42dd9555eb177f35150cf9aa240b61e5"
+
+FILESEXTRAPATHS:prepend := "${THISDIR}/files/ae350-ax45mp:"
+
+inherit autotools-brokensep deploy
+
+PV = "0.9+git${SRCPV}"
```

Comment:
Why do we need to use an outdated version of OpenSBI? Can't we just use the version in upstream?